src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v1/TransactionClosePaymentRetryQueueConsumer.kt

```diff
@@ -168,17 +166,16 @@ class TransactionClosePaymentRetryQueueConsumer(
                 "Got exception while retrying closePaymentV2 for transaction with id ${tx.transactionId}!",
                 exception)
 
+              val refundTransaction: Boolean
```

Comment:
Refactor this to return a triple in order to avoid refund transaction val assignation into if/else statement.

something like:
```code
val (statusCode, errorDescription, refundTransaction) =
...
Triple(exception.statusCode, exception.errorResponse?.description,true)
...
```